### PR TITLE
feat(ui): added hero component to ui

### PR DESCRIPTION
**Pull request by JimBurch**
image
image
image
image

### CODE REVIEW

File: packages/core/src/types/index.ts [hunk 1]
```diff
@@ -178,7 +178,8 @@ export interface TypographyProps {
     | "heading4"
     | "heading5"
     | "heading6"
-    | "span";
+    | "span"
+    | string;
```
Comment (erwstout) [inline packages/core/src/types/index.ts:182]: why the addition of `string` here?

**Review by erwstout — CHANGES_REQUESTED** [review]
SOLID DUDE! Just some small stuff.

File: packages/core/src/types/index.ts (hunk 1 above)
Comment (JimBurch) [reply packages/core/src/types/index.ts:182]: turns out I didn't need that after I typed out the react state... removed

File: packages/core/src/Hero/index.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      height: "100%",
+    },
+  },
+  column: {
+    display: "flex",
+    flexWrap: "wrap",
+    alignItems: "center",
+    justifyContent: "center",
+    width: "100%",
+    "& > *": {
```
[the remark this answers is not in the document]
Comment (JimBurch) [reply packages/core/src/Hero/index.tsx:35]: the dynamic rendering opens up a few possibilities but I specified them instead of a *

**Review by erwstout — APPROVED** [review]

**Comment by erwstout** [comment]
@JimBurch I un-approved because we need Jest Tests and also Story book stories before this can be merged!

**Review by denvermullets — APPROVED** [review]

**Review by erwstout — APPROVED** [review]

**Pull request merged**